### ISSUE TITLE
TensorFlow 2.17 for macOS x86_64 is not on PyPI

### ISSUE THREAD

**Issue by Craigacp** [tensorflow/tensorflow#75279]
### Issue type

Build/Install

### Have you reproduced the bug with TensorFlow Nightly?

No

### Source

binary

### TensorFlow version

2.17.0

### Custom code

No

### OS platform and distribution

macOS x86_64

### Python version

3.11

### Current behavior?

TensorFlow 2.17 did not publish macOS x86_64 binaries to PyPI, only arm64 binaries. In TensorFlow-Java we unpack the PyPI binaries so we can repackage them in our Java builds, but there was no notice of the removal of macOS x86_64 binaries in the 2.16 or 2.17 release notes.

### Standalone code to reproduce the issue

```shell
`pip install tensorflow==2.17.0` on a macOS x86_64 machine.
```

**Comment by Venkat6871** [comment]
@learning-to-play .

**Comment by learning-to-play** [comment]
See [TensorFlow 2.16 release notes](https://github.com/tensorflow/tensorflow/releases/tag/v2.16.1): "Mac x86 users: Mac x86 builds are being deprecated and will no longer be released as a Pip package from TF 2.17 onwards."

**Comment by Craigacp** [comment]
Ok thanks, I missed that in the notes.
